ionic2/app/services/api.service.ts

```diff
@@ -74,6 +74,16 @@ export class ApiService {
   }
 
   /**
+   * Get Pokemon by description.
+   * @param {string} description - Description of the Pokemon
+   * @returns {Observable<Pokemon[]>}
+   */
+  getPokemonByDescription(description: string): Observable<Pokemon[]> {
```

Comment:
@AlexanderLill See edit in the comment above, I'm not going to hold this back, but I'd suggest to hold back anyway :wink: